roles/contiv/tasks/pkgMgrInstallers/centos-install.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: PkgMgr CentOS | Install net-tools pkg for route
+  yum:
+    pkg=net-tools
+    state=latest
+
+- name: PkgMgr CentOS | Get openstack kilo rpm
+  get_url:
+    url: https://repos.fedorapeople.org/repos/openstack/openstack-kilo/rdo-release-kilo-2.noarch.rpm
+    dest: /tmp/rdo-release-kilo-2.noarch.rpm
+    validate_certs: False
+  environment:
+    http_proxy: "{{ http_proxy|default('') }}"
+    https_proxy: "{{ https_proxy|default('') }}"
+    no_proxy: "{{ no_proxy|default('') }}"
+  tags:
+    - ovs_install
+
+- name: PkgMgr CentOS | Install openstack kilo rpm
+  yum: name=/tmp/rdo-release-kilo-2.noarch.rpm state=present
+  tags:
+    - ovs_install
+
+- name: PkgMgr CentOS | Install ovs
+  yum:
+    pkg=openvswitch
+    state=latest
+  environment:
+    http_proxy: "{{ http_proxy|default('') }}"
+    https_proxy: "{{ https_proxy|default('') }}"
+    no_proxy: "{{ no_proxy|default('') }}"
+  tags:
+    - ovs_install
```

Comment:
Agreed, we will move to the ovs installed by openshift in future. 

However currently we have done a lot of testing with ovs 2.5 which we install currently as part of contiv installation). This gives us confidence in the control plane + data plane combination that we have tested thoroughly and our QA automated testing suite also uses. So we would like to go ahead with our OVS for now and not hold up this commit/ merge. Once we have had sufficient experience testing with ovs 2.4 or 2.6 as deployed through openswitch, we will definitely switch to that version and not install our own ovs 2.5.